src/CommandingOfficers/Modifiers/CODefenseModifier.java

```diff
@@ -1,13 +1,9 @@
 package CommandingOfficers.Modifiers;
 
-import Units.UnitModel;
+import Engine.Combat.StrikeParams.BattleParams;
+import Engine.UnitMods.UnitModifierWithDefaults;
 
-import java.util.ArrayList;
-
-import CommandingOfficers.Commander;
-import CommandingOfficers.Modifiers.COModifier.GenericCOModifier;
-
-public class CODefenseModifier extends GenericCOModifier
+public class CODefenseModifier implements UnitModifierWithDefaults
```

Comment:
Should this (and others) be renamed to e.g. `UnitDefenseModifier` for consistency?